app/templates/view_buyers.html

```diff
@@ -0,0 +1,85 @@
+{% import "toolkit/summary-table.html" as summary %}
+
+{% extends "_base_page.html" %}
+
+{% block page_title %}
+    Buyers – Digital Marketplace admin
+{% endblock %}
+
+{% block breadcrumb %}
+  {%
+      with items = [
+          {
+              "link": url_for('.index'),
+              "label": "Admin home"
+          },
+      ]
+  %}
+    {% include "toolkit/breadcrumb.html" %}
+  {% endwith %}
+{% endblock %}
+
+{% block main_content %}
+  {% with messages = get_flashed_messages(with_categories=true) %}
+  {% if messages %}
+      {% for category, message in messages %}
+          {% if message == 'no_brief' %}
+              {% set displayed_message = "Sorry, we couldn't find a brief with the supplied ID: {}".format(brief_id) %}
```

Comment:
Small point, but I's drop the word `supplied` from this message and just say `...couldn't find a brief with the ID`.  

Testing it locally it threw me a bit - I read it as `supplier ID` on the page, an easy mistake to make as we have supplier IDs all over the place.